src/SymptomHistory/symptom.ts

```diff
@@ -0,0 +1,133 @@
+export type Symptom =
+  | "chest_pain_or_pressure"
+  | "difficulty_breathing"
+  | "lightheadedness"
+  | "disorientation_or_unresponsiveness"
+  | "fever"
+  | "chills"
+  | "cough"
+  | "loss_of_smell"
+  | "loss_of_taste"
+  | "loss_of_appetite"
+  | "vomiting"
+  | "diarrhea"
+  | "body_aches"
+  | "other"
+
+export const all: Symptom[] = [
+  "cough",
+  "fever",
+  "chest_pain_or_pressure",
+  "difficulty_breathing",
+  "lightheadedness",
+  "disorientation_or_unresponsiveness",
+  "chills",
+  "loss_of_smell",
+  "loss_of_taste",
+  "loss_of_appetite",
+  "vomiting",
+  "diarrhea",
+  "body_aches",
+  "other",
+]
+
+export const fromString = (rawSymptom: string): Symptom | null => {
+  switch (rawSymptom) {
+    case "chest_pain_or_pressure": {
+      return "chest_pain_or_pressure"
+    }
+    case "difficulty_breathing": {
+      return "difficulty_breathing"
+    }
+    case "lightheadedness": {
+      return "lightheadedness"
+    }
+    case "disorientation_or_unresponsiveness": {
+      return "disorientation_or_unresponsiveness"
+    }
+    case "fever": {
+      return "fever"
+    }
+    case "chills": {
+      return "chills"
+    }
+    case "cough": {
+      return "cough"
+    }
+    case "loss_of_smell": {
+      return "loss_of_smell"
+    }
+    case "loss_of_taste": {
+      return "loss_of_taste"
+    }
+    case "loss_of_appetite": {
+      return "loss_of_appetite"
+    }
+    case "vomiting": {
+      return "vomiting"
+    }
+    case "diarrhea": {
+      return "diarrhea"
+    }
+    case "body_aches": {
+      return "body_aches"
+    }
+    case "other": {
+      return "other"
+    }
+    default: {
+      return null
+    }
+  }
+}
+
+export const toString = (symptom: Symptom): string => {
+  switch (symptom) {
+    case "chest_pain_or_pressure": {
+      return "chest_pain_or_pressure"
+    }
+    case "difficulty_breathing": {
+      return "difficulty_breathing"
+    }
+    case "lightheadedness": {
+      return "lightheadedness"
+    }
+    case "disorientation_or_unresponsiveness": {
+      return "disorientation_or_unresponsiveness"
+    }
+    case "fever": {
+      return "fever"
+    }
+    case "chills": {
+      return "chills"
+    }
+    case "cough": {
+      return "cough"
+    }
+    case "loss_of_smell": {
+      return "loss_of_smell"
+    }
+    case "loss_of_taste": {
+      return "loss_of_taste"
+    }
+    case "loss_of_appetite": {
+      return "loss_of_appetite"
+    }
+    case "vomiting": {
+      return "vomiting"
+    }
+    case "diarrhea": {
+      return "diarrhea"
+    }
+    case "body_aches": {
+      return "body_aches"
+    }
+    case "other": {
```

Comment:
🔥 🧯

Reply:
For the benefit of future readers, I would like to note that the above comment indicates that this refactor is, in fact, fire.